### PR TITLE
Added Card Hover Effect files

### PR DESCRIPTION
**Pull request by shaikh-7abish**

### CODE REVIEW

File: tasks-by-users/shaikh-7abish/04-card-hover-effect/main.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+.container .card .face.face2 .content a{
+    margin-top: 10px;
+    display: inline-block;
+    text-decoration: none;
+    padding: 5px;
+    font-weight: 400;
+    color: #000;
+    border: 1px solid #000;
+    border-radius: 5px;
```
Comment (coder-mezab) [inline tasks-by-users/shaikh-7abish/04-card-hover-effect/main.css:84]: grp these properties @shaikh-7abish

File: tasks-by-users/shaikh-7abish/04-card-hover-effect/main.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    font-size: 1.3rem;
+}
+
+.container .card .face.face2{
+    background: #fff;
+    padding: 20px;
+    box-sizing: border-box;
+    box-shadow: 0 20px 50px rgba(0,0,0,0.8);
+    opacity: 1;
+    transform: translateY(-12rem);
```
Comment (coder-mezab) [inline tasks-by-users/shaikh-7abish/04-card-hover-effect/main.css:64]: group similar properties

**Review by anask601 — APPROVED** [review]

**Review by AbbasShaikh0304 — APPROVED** [review]

**Review by narayand16 — APPROVED** [review]

File: tasks-by-users/shaikh-7abish/04-card-hover-effect/main.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
+body{
```
Comment (faran4engg) [inline tasks-by-users/shaikh-7abish/04-card-hover-effect/main.css:1]: looks like `prettier` is not configured

**Review by faran4engg — CHANGES_REQUESTED** [review]
Let's fix these comments !

**Review by AlishaPrasad — APPROVED** [review]

**Comment by faran4engg** [comment]
Live demo link ?

**Review by faran4engg — APPROVED** [review]

File: tasks-by-users/shaikh-7abish/04-card-hover-effect/main.css (hunk 3 above)
Comment (faran4engg) [reply tasks-by-users/shaikh-7abish/04-card-hover-effect/main.css]: Resolving conversation

**Pull request merged**